### PR TITLE
chore: Disabling failing test 

### PR DESCRIPTION
**Pull request by FarrukhMasud**
## Related Issues/PRs

Disabling a Rapids unit tests as it has been failing, causing the build to be marked as failed. We will fix this unit test, in the meantime, we are disabling this test

## How is this patch tested?
Trigger unit test to validate everything runs as expected.

## Does this PR change any dependencies?

- [X] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [ ] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by FarrukhMasud** [comment]
This is not correct, closing the PR

File: pipeline.yaml [hunk 1]
```diff
@@ -143,8 +143,6 @@ jobs:
         TEST-CLASS: "com.microsoft.azure.synapse.ml.nbtest.DatabricksCPUTests"
       databricks-gpu:
         TEST-CLASS: "com.microsoft.azure.synapse.ml.nbtest.DatabricksGPUTests"
-      databricks-rapids:
-        TEST-CLASS: "com.microsoft.azure.synapse.ml.nbtest.DatabricksRapidsTests"
```
[the remark this answers is not in the document]
Comment (FarrukhMasud) [reply pipeline.yaml:147]: I have added comments and commented it out

**Review by mhamilton723 — APPROVED** [review]

**Pull request merged**